### ISSUE TITLE
Waypoints labelled numerically incorrectly 

### ISSUE THREAD

**Issue by 41Mike304** [dcs-liberation/dcs_liberation#3037]
The waypoints in Liberation planner start at WP1 where WP0 should be, although they present correctly when in the aircraft (F18). Quite a confusing system in practice as this is vital information that is very easy to mislabel when writing a brief. Also screenshots of flight plans need to be edited and labelled separately in a different program before being presented to the flight.

**Comment by DanAlbert** [comment]
I've got no idea what you're talking about. Provide some screenshots or something and show what you think is wrong.

**Comment by 41Mike304** [comment]
I’m away from my PC until Thursday, I can explain though. Create any Liberation flight plan and the first waypoint you arrive at after take off in the Liberation planner will be waypoint 2 (hold) where it should be waypoint 1 as it is in the aircraft.

**Comment by Starfire13** [comment]
I was chatting with 41Mike304 about this on discord earlier and I think I understand what he means.

You can see here how Liberation numbers the waypoints on this flight from 1 to 9. So, take off is 1, hold is 2, so on and so forth.

![Screenshot 2023-06-26 222357](https://github.com/dcs-liberation/dcs_liberation/assets/72491792/1d28acf7-5687-43d0-b0c3-b2cf8d2073c9)

DCS starts the waypoint number from 0. This means take off is 0, hold is 1, etc.

![Screenshot 2023-06-26 222628](https://github.com/dcs-liberation/dcs_liberation/assets/72491792/116ddbd8-e17b-4fc2-a307-590b7517998e)

Likewise on the kneeboard, the waypoint numbers in Liberation don't match the waypoint numbers in DCS.
![Screenshot 2023-06-26 222731](https://github.com/dcs-liberation/dcs_liberation/assets/72491792/589ddb03-f26f-4fb4-b615-5d64c5e0d21e)

I think it would be beneficial for some of those playing multiplayer to have consistent waypoint numbering between Liberation and DCS.

**Comment by 41Mike304** [comment]
Many thanks for the detailed explanation Starfire.

**Comment by 41Mike304** [comment]
I should have added, the Kneeboard actually does number the waypoints correctly, it’s the mission planner that doesn’t.

**Comment by Starfire13** [comment]
Yes. Essentially, everything viewed in DCS is correct (even the Liberation generated kneeboard pages). It's just the Edit Flight window in Liberation that is using a different numbering system from DCS because it's starting the count on 1 instead of 0.

**Comment by DanAlbert** [comment]
Thanks. There are at least three places that waypoints are numbered, so I had no idea which was meant.

**Comment by DanAlbert** [comment]
Looks like it's wrong in the Liberation map as well: 
![image](https://github.com/dcs-liberation/dcs_liberation/assets/315852/3f8beaf4-0528-4350-9865-5959625c7bcb)

Sort of an aside, but is it universally true that waypoints start at 0 in DCS? I think it is for all the planes I fly, but that's a small set and I've never really thought about it. If that's not the case the kneeboards should probably have aircraft specific waypoint naming. There are more complicated cases like the F-14 (and Viggen?) where some waypoints have names rather than numbers, and the Apache has several types of points (but I think the ME can only control one kind, and those would just need the `W` prefix). Not the topic here, but maybe worthy of another FR.

**Comment by DanAlbert** [comment]
https://github.com/dcs-liberation/dcs_liberation/pull/3042. This is actually somewhat new. The game data used to not actually include a waypoint for takeoff, so there were a few places that either injected one (the map was actually seeing two takeoff waypoints), or ignored its absence (like the flight dialog). We added the data to the model at some point but forgot to clean up the workarounds (which is why when I read this report I was surprised; last time I looked at that dialog or zoomed in closely enough to see the labels was probably before that change 🙂)
